proto/forch_configuration.proto

```diff
@@ -237,3 +240,11 @@ message DataplaneMonitoring {
   // threshold of number of packets per second for each vlan; indexed by vlan
   map<int32, int32> vlan_pkt_per_sec_thresholds = 2;
 }
+
+/*
+ * Parameters for Faucet config porcessing
+ */
+message FaucetConfigProcessing {
+  // maximum retry times for Faucet config hash checking
+  int32 config_hash_max_retry = 1;
```

Comment:
can/should this be put in the EventClientConfig message?  Seems like it's conceptually related...

Reply:
Yes, moved to EventClientConfig..